modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -253,9 +254,17 @@
   [field unit]
   (if (= unit :default)
     field
-    (let [column field]
+    (let [date-trunc? (-> (get-mongo-version)
```

Comment:
I think naming this something like `supports-date-trunc?` or `supports-dateTrunc?` might be a little bit clearer. Otherwise I can see this below and think it means "are we doing a date trunc?" or something like that